### PR TITLE
v3.2.0

### PR DESCRIPTION
**Pull request by CleanBread**
deBridge feature

### CODE REVIEW

File: src/hooks/useDeBridgeBet.ts [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+              {
+                affiliate,
+                minOdds: rawMinOdds,
+                data: betData,
+              },
+            ],
+          }),
+        },
+      }),
+      referralCode: '9126',
```
Comment (on47sky) [inline src/hooks/useDeBridgeBet.ts]: dynamic prop

File: src/hooks/useDeBridgeSupportedTokens.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+type Props = {
+  chainId: number
+  enabled?: boolean
+}
+
+export const useDeBridgeSupportedTokens = ({ chainId, enabled = true }: Props) => {
+  const queryFn = async () => {
+    const response = await fetch(`${deBridgeUrl}/token-list?chainId=42161`)
```
Comment (on47sky) [inline src/hooks/useDeBridgeSupportedTokens.ts]: pass `chainId` query param from props

File: src/hooks/useDeBridgeSupportedTokens.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    return {
+      supportedTokens,
+      tokenAddresses: supportedTokens.map(({ address }) => address),
+    }
+  }
+
+  const { isFetching, data, refetch } = useQuery({
+    queryKey: [ '/debridge-supported-tokens', chainId ],
+    queryFn,
+    enabled,
```
Comment (on47sky) [inline src/hooks/useDeBridgeSupportedTokens.ts:38]: mb `enabled: Boolean(chainId && enabled)`?

**Review by on47sky — CHANGES_REQUESTED** [review]

File: src/hooks/useDeBridgeBet.ts (hunk 1 above)
Comment (CleanBread) [reply src/hooks/useDeBridgeBet.ts]: 344bd4aa812c217f7ae2ac904a407107740f1235

File: src/hooks/useDeBridgeSupportedTokens.ts (hunk 2 above)
Comment (CleanBread) [reply src/hooks/useDeBridgeSupportedTokens.ts]: 8aeed0c686805f9ab7bdd47cc30f9d7598cfd974

File: src/hooks/useDeBridgeSupportedTokens.ts (hunk 3 above)
Comment (CleanBread) [reply src/hooks/useDeBridgeSupportedTokens.ts:38]: chainId is required prop

**Review by on47sky — APPROVED** [review]

**Pull request merged**